config/src/config.rs

```diff
@@ -330,7 +330,7 @@ pub struct Config {
     pub hide_tab_bar_if_only_one_tab: bool,
 
     #[serde(default)]
-    pub enable_scroll_bar: bool,
+    pub scroll_bar_mode: ScrollBarMode,
```

Comment:
For backwards compatibility purposes, it would be nice to allow `enable_scroll_var = true` to continue working when folks upgrade to a version with these changes.

I'd propose adding:

```rust
#[serde(default)]
pub enable_scroll_bar: Option<bool>,
```

and then adding a helper method:

```
pub fn get_scroll_bar_mode(&self) -> ScrollBarMode {
   match self.enable_scroll_bar {
       Some(true) => ScrollBarMode::AllPanes,
       Some(false) => ScrollBarMode::None,
       None => self.scroll_bar_mode,
   }
}
```
}

Reply:
Good idea. Done.